blocks/lists.js

```diff
@@ -133,7 +133,7 @@ Blockly.Blocks['lists_create_with'] = {
    */
   init: function() {
     this.setHelpUrl(Blockly.Msg['LISTS_CREATE_WITH_HELPURL']);
-    this.setColour(Blockly.Msg['LISTS_HUE']);
+    this.setStyle("lists");
```

Comment:
nit: Strings in JS code should use single quotes here and below (but leave the ones in the JSON blobs as double quotes).